docs/setup-and-installing.md

```diff
@@ -83,20 +83,20 @@ If you are using Rider, install the [**Unity Support** plugin](https://github.co
 
 <%(/Expandable)%>
 
-### Validate your installation
```

Comment:
removing the validate section as `spatial diagnose` is useless for us. added some extra links for how to get help instead